app/src/main/java/MainActivity.java

```diff
@@ -1,26 +0,0 @@
-import android.location.LocationManager;
-import android.widget.ImageView;
-import android.widget.RelativeLayout;
-import android.widget.TextView;
-
```

Comment:
File seemed to not be in use so I removed it.

Rationale: Files that aren't used should be in the project as they provide clutter.